### PR TITLE
improve apt commands

### PR DESCRIPTION
**Pull request by diamant3**
- use `apt` instead of `apt-get` and remove install/update confirmation
- fix `Unable to locate package libusb-0.1`

### CODE REVIEW

File: quickstart.md [hunk 1]
```diff
@@ -27,11 +27,11 @@ wsl --install
 Now run bash and run the following commands to install the dependencies:
 
 ```shell
-sudo apt-get update
+sudo apt update -y
```
Comment (sharkwouter) [inline quickstart.md]: `-y` should not be required for update

**Comment by sharkwouter** [comment]
I'm confused, the package libusb-0.1 exists for all supported versions of Ubuntu. The specific version of the package should not matter. Which Ubuntu version are you seeing this error on?

File: quickstart.md (hunk 1 above)
Comment (diamant3) [reply quickstart.md]: my bad i forgot that is for installation only, let me update that

**Comment by diamant3** [comment]
here's my `lsb_release -a` output, wouter :(

```
diamant3@MSI:~$ lsb_release -a
LSB Version:	core-11.1.0ubuntu4-noarch:printing-11.1.0ubuntu4-noarch:security-11.1.0ubuntu4-noarch
Distributor ID:	Ubuntu
Description:	Ubuntu 22.04.4 LTS
Release:	22.04
Codename:	jammy
```

edit: sorry i forgot, this is not WSL but i think they are the same right? wsl ubuntu and the ubuntu linux? :(

**Comment by diamant3** [comment]
the culprit is the package manager, `apt` is newer that's why unable to locate the `libusb-0.1` package.

edit: I will close this PR because The `apt-get` is much more popular than `apt` in terms of tutorials and i think we don't need to enforce the removal of install confirmation for better user experience :)

**Pull request closed without merging**